.github/workflows/publish-requirements.yml

```
@@ -37,7 +35,9 @@ jobs:
 
       - name: Install dependencies
         run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2261:error:3:27: Multiple redirections compete for stdout. Use cat, tee, or pass filenames instead

(shellcheck)